stdlib/3/itertools.pyi

```diff
@@ -97,6 +97,8 @@ def product(iter1: Iterable[Any],
             *iterables: Iterable[Any]) -> Iterator[Tuple[Any, ...]]: ...
 @overload
 def product(*iterables: Iterable[Any], repeat: int) -> Iterator[Tuple[Any, ...]]: ...
```

Comment:
Does just adding `= ...` to `repeat` here work? I feel like this and the next overload would be overlapping.